test/fixtures.js

```diff
@@ -114,3 +120,9 @@ export const createUser = (overrides = {}) => ({
 export const createSubjectDayData = (overrides = {}) => ({
   ...overrides,
 })
+
+export const createConfigurationArrayData = (overrides = []) => [
+  { name: 'config1', owner: 'owl' },
+  { name: 'config2', owner: 'eagle' },
+  ...overrides,
+]
```

Comment:
we should just have this be a `createConfiguration` factory and the tests that need multiple can create multiple. What other fields does a configuration have? The factories should set up a base entity with all fields.

Reply:
[1dcf49f](https://github.com/AMP-SCZ/dpdash/pull/305/commits/1dcf49fc62eb980f67c8ed28fed3c7a2b4109eec)
[e0d3c3d](https://github.com/AMP-SCZ/dpdash/pull/305/commits/e0d3c3d035b96a8af84e42d5a9680aee6c6ea853)